packages/turf-isobands/.npmignore

```diff
@@ -0,0 +1,2 @@
+test
+coverage
```

Comment:
Not needed since this is covered in the `package.json` using `"files": ["index.js", "index.d.ts"]`